### ISSUE TITLE
Additional space in history string

### ISSUE THREAD

**Issue by durack1** [PCMDI/cmor#231]
For the [obs4MIPs demo](https://github.com/PCMDI/obs4MIPs-cmor-tables/tree/master/demo) the history attribute written has an additional space between "and  requirements", e.g.
```
:history = "2017-09-14T21:33:44Z CMOR rewrote data to be consistent with CF standards and  requirements." ;
```
The issue comes from the string concatenation found [here](https://github.com/PCMDI/cmor/blob/master/Src/cmor.c#L2667) - `msg` is missing with `obs4MIPs`

**Comment by dnadeau4** [comment]
So what is mip_era for obs4MIPs?

**Comment by durack1** [comment]
@dnadeau4 it's no longer a field - see https://goo.gl/jVZsQl

**Comment by dnadeau4** [comment]
I cannot delete it for CMIP6, but can set it to this for other projects.
:history = "2017-09-14T21:33:44Z CMOR rewrote data to be consistent with CF standards.";

**Comment by durack1** [comment]
@dnadeau4 perfect.. If I was to add a `mip_era = obs4MIPs` into the input json, the string that I would get would be:
```
:history = "2017-09-14T21:33:44Z CMOR rewrote data to be consistent with CF standards and obs4MIPs requirements." ;
```
Is this correct?

**Comment by dnadeau4** [comment]
Yes it would be.

**Comment by dnadeau4** [comment]
fix in 3.2.7
